packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx

```diff
@@ -0,0 +1,55 @@
+import {MenuItem, TextField} from "@material-ui/core"
+import {getLanguages} from "lib/contentful-api"
```

Comment:
Has this been added to the dependencies?

Reply:
Just fixed this. Good spot.